src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -238,6 +250,18 @@ private void styleFetchedListView(CheckListView<CitationRelationItem> listView)
                                 }
                         });
                         vContainer.getChildren().add(jumpTo);
+
+                        Button compareButton = IconTheme.JabRefIcons.MERGE_ENTRIES.asButton();
+                        compareButton.setTooltip(new Tooltip(Localization.lang("Compare with existing entry")));
+                        compareButton.setOnMouseClicked(event -> {
+                            openPossibleDuplicateEntriesWindow(entry);
+
+                            // update local entry of selected citation relation item
+                            listView.getItems().set(listView.getItems().indexOf(entry), new CitationRelationItem(entry.entry(), entriesMerge, true));
+                            // let main table know this is a citation merge
+                            libraryTab.getMainTable().setCitationMerge(true);
```

Comment:
Wrong position - because if I click "cancel" on the merge, the mode is not reset - I show you a better position in the code below.